frontend/amundsen_application/static/js/pages/TableDetailPage/TableHeaderBullets/index.tsx

```diff
@@ -51,13 +51,15 @@ export class TableHeaderBullets extends React.Component<TableHeaderBulletsProps>
           <li>{getDisplayNameByResource(ResourceType.feature)}</li>
         )}
         {database !== 'feature' && (
-          <li>{getDisplayNameByResource(ResourceType.table)}</li>
+          <>
```

Comment:
need to have this reach fragment component https://reactjs.org/docs/fragments.html
otherwise eslint will fail (a type check)